README.md

```diff
@@ -21,8 +21,11 @@ There is a 500 sUSD bounty for proposing a SIP that reaches the 'Implemented' ph
 
```

Comment:
FYI: You should include `.DS_Store` into `.gitignore` if you use Finder for these folders.